README.txt

```diff
@@ -76,9 +76,9 @@ In order to install Jira, we will need a working version of Java, a dedicated us
 
  1. Set the jira home directory by manipulating some Jira files.
       nano $JIRA_BINARIES/atlassian-jira/WEB-INF/classes/jira-application.properties
-      edit: jira.home = "~jira/jira/jira_home"
+      edit: jira.home = "/home/jira/jira_home"
       nano $JIRA_BINARIES/bin/setenv.sh
-	  edit: JIRA_HOME="~jira/jira/jira_home"
```

Comment:
I actually intended to write the jira folder inside the jira user's home directory, but I must admit it's confusing.

Reply:
It's not confusing, your path is totally fine. The problem is that Jira expects an absolute path as JIRA_HOME, but `~jira/jira/jira_home` is relative. So I wasn't able to start the installation with that path, so that's why I replaced your path by the absolute one. I did not create a separate folder "jira" inside the jira-user's home directory as I did not see why it would be necessary, but that wasn't the point of the change here.

Reply:
Oh, I missed this at first, thank you for the explanation.